pandas/core/ops/__init__.py

```diff
@@ -364,11 +369,57 @@ def dispatch_to_series(left, right, func, str_rep=None, axis=None):
     """
     # Note: we use iloc to access columns for compat with cases
     #       with non-unique columns.
+    eval_kwargs = eval_kwargs or {}
+
     import pandas.core.computation.expressions as expressions
 
     right = lib.item_from_zerodim(right)
+
     if lib.is_scalar(right) or np.ndim(right) == 0:
 
+        new_blocks = []
```

Comment:
I would rather actually actualy do this in the block manager no? (maybe put in internals/ops.py) we should actually move *all* block type ops there (e.g. from groupby as well). OR isolate this code in ops somewhere (maybe internals.py), so things are separated.

Reply:
Generally agree on getting Block/BlockManager-touching things isolated in internals.  Will give some thought to how/when to move this.  ATM this is in Proof of Concept phase while I figure out how to handle the remaining cases.